packages/core-types/src/SharedPasswordProtection.ts

```diff
@@ -2,9 +2,37 @@ import { ISerializable, Serializable, serialize, validate } from "@js-soft/ts-se
 import { CoreBuffer, ICoreBuffer } from "@nmshd/crypto";
 import { CoreError } from "./CoreError";
 
+type Enumerate<N extends number, Acc extends number[] = []> = Acc["length"] extends N ? Acc[number] : Enumerate<N, [...Acc, Acc["length"]]>;
+type IntRange<From extends number, To extends number> = Exclude<Enumerate<To>, Enumerate<From>>;
+
+export enum PasswordLocationIndicatorMedium {
+    RecoveryKit = "RecoveryKit",
+    Self = "Self",
+    Letter = "Letter",
+    RegistrationLetter = "RegistrationLetter",
+    Mail = "Mail",
+    Sms = "Sms",
```

Comment:
The same applies to each occurrence.

```suggestion
    SMS = "SMS",
```

Reply:
Mhh subjectively I prefer `Sms` over `SMS`. It matches our variable casing, although SMS is correct in natural language. Can we say which casing is better regarding some objective measure?

Reply:
SMS is correct, but you can also use sms

Reply:
Okay, I'll change the enum value to SMS. What about the string? Should we allow any casing? Same for the other strings, i.e. should an integrator be allowed to write "letter" and "Letter"?